src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs

```diff
@@ -412,5 +412,11 @@ public static class Descriptors
 
 		public static DiagnosticDescriptor PX1095_PXDBScalarMustBeAccompaniedNonDBTypeAttribute { get; } =
 			Rule("PX1095", nameof(Resources.PX1095Title_PXDBScalar).GetLocalized(), Category.Default, DiagnosticSeverity.Error, DiagnosticsShortName.PX1095PXDBScalar);
+
+		public static DiagnosticDescriptor PX1096_PXOverrideMustMatchSignature { get; } =
+			Rule("PX1096", nameof(Resources.PX1096Title_SignatureMismatch).GetLocalized(), Category.Default, DiagnosticSeverity.Warning, DiagnosticsShortName.PX1096SignatureMismatch);
+
+		public static DiagnosticDescriptor PX1096_PXOverrideContainerIsNotPXGraphExtension { get; } =
```

Comment:
I'm not sure that this check is needed. We don't add checks for everything just because it can be checked since every diagnostic takes developer machine resources and slows down IDE a bit. 

I will discuss it with the ISV team but I would advise to remove this check. Supporting PXOverride lookup in every class will force you to design an analyzer that have to check type declarations in code instead of graph extensions only. This is bad for performance. If this scenario is really useful and encountered by our developers then it's still better to split the analyzer into two analyzers - an aggregated graph analyzer that will check only graph extensions for the correct signature and the standard Acumiantor roslyn analyzer derived from `PXDiagnosticAnalyzer` that will check the compilation root node, collect all "PXOverride" attributes from it and check its containing types.

Reply:
@taskinkemal I've discussed with ISVs the `PXOverride` scenario, it seems that this check can be useful for them. In this case it should be done in a separate analyzer.

Reply:
With respect to another comment above, maybe it's best to exclude this rule from this commit and keep it as simple as possible. Later we can add this analyzer as a separate commit.

Reply:
I'm not sure I understand what you mean by commit. If you mean a separate PR with implementation of that analyzer than yes, I agree. It would be cleaner to have a separate PR for that although you would probably need to remove related unit tests from this PR.

If you are talking about just a separate git commits inside this branch then please make more granular git commits. It shouldn't be one git commit per PR.

Reply:
That's true. Sorry, I meant PR. I've removed the related unit tests already. Thank you.